text/0265-event-vision.md

```diff
@@ -0,0 +1,75 @@
+# Event Basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to document and agree on a few basics.
+
+### What are OpenTelemetry Events?
+
+OpenTelemetry Events are a type of OpenTelemetry Log that requires an event name and follows a specific structure implied by that event name.
+
+They are a core concept in OpenTelemetry Semantic Conventions.
+
+### OTLP
+
+Since OpenTelemetry Events are a type of OpenTelemetry Log, they share the same OTLP log data structure and pipeline.
+
+### API
+
+OpenTelemetry should provide a (user-facing) Log API that includes the capability to emit OpenTelemetry Events.
+
+### Interoperability with Generic Logging Libraries
+
+It should be possible to send OpenTelemetry Logs from the OpenTelemetry Log API to a generic logging library (e.g., Log4j).
```

Comment:
From a customer point of view, I think it may be confusing, for example, when to use OTel Logging API mapped to Log4j vs when to use Log4j directly. It also seems contrary to what OTel recommended before, which is "continue to use your language logging library". Should we be explicit "continue to use your existing logging APIs except when you need more structure, etc."?

Reply:
I would like us to back away from passing judgements on these APIs. We should definitely explain the use cases: appending logs from application loggers, and writing OTel instrumentation. And it is worth mentioning that events can be created using any method on our logger, so application developers can choose to integrate the OTel conventions into their application logging via their log appender.

But I don't think there is a need to recommend that developers should stay away from the API. That feels like a very confusing recommendation.

Reply:
Specifically regarding the case where users want to loop Otel logs through their own logger and back to OTel – I'm not really sold on this being a common use case. Which means that we need to support it, but we don't need to make a big deal about it or make it front and center.

The common use case presented for wanting to loop the logs back through the application logger is to export them  to other log appenders. But it's easy to add more exporters to OTel, so this use case doesn't seem like much of a barrier in the long term.

As far as other kinds of log processing are concerned, mutating Otel events is actively bad and unhelpful. It feels unlikely that many developers will want to do this. Again, we can support it, but users with this problem will self identify and find the docs for it, we don't need to make this use case a top level concept.